docs/power-to-energy.md

```diff
@@ -0,0 +1,93 @@
+# Power-to-Energy
+
+`power-to-energy` is a plugin for calculating enegy consumption (kWh) from wattage and duration.
+
+You provide the wattage value and the duration.
+
+For example, you could add `wattage` and `duration`. The `energy-consumption` would then be added to every observation in your input array as the calculated value of `wattage` and `duration`.
```

Comment:
I think both `wattage` and `duration` are mandatory parameters, and these sentences are similar with line 3. Thus we might remove them.
```suggestion
```